test/pytest/test_qkeras.py

```diff
@@ -480,6 +481,109 @@ def test_quantised_po2_bit_width(backend, io_type, strategy):
     np.testing.assert_allclose(y_hls.flatten(), y_keras.flatten(), rtol=2e-2)
 
 
+@pytest.mark.parametrize('backend', ['Quartus'])
+def test_qsimplernn(backend):
+    '''
+    Test proper handling of QSimpleRNN.
+    '''
+    set_random_seed(42)
```

Comment:
Generally you should not set the seed. We use pytest-randomly to set the seed. If there is sensitivity to random numbers, maybe the bitwidths or rounding/saturation need to be changed?

Reply:
Fixed by increasing the bitwidth and reducing the `linspace` range .